Doc/library/sqlite3.rst

```diff
@@ -47,84 +47,152 @@ This document includes four main sections:
       PEP written by Marc-André Lemburg.
 
 
+.. We use the following practises for SQL code:
+   - UPPERCASE for keywords
+   - SnakeCase for schema
+   - single quotes for string literals
+   - singular for table names
+   - if needed, use double quotes for table and column names
+
 .. _sqlite3-tutorial:
 
 Tutorial
 --------
 
-To use the module, start by creating a :class:`Connection` object that
-represents the database.  Here the data will be stored in the
-:file:`example.db` file::
+In this tutorial, you will learn the basics of the :mod:`!sqlite3` API
+by creating an on-disk movie database :file:`tutorial.db`
+about Monty Python movies.
+It assumes a fundamental understanding of database concepts,
+including `cursors`_ and `transactions`_.
+
+First, create a new database to hold the movie data,
+and open a database connection to allow :mod:`!sqlite3` to work with it.
+Call the :func:`sqlite3.connect` function to
+to create a connection to the database :file:`tutorial.db`,
+implicitly creating it in the current working directory if it does not exist::
 
    import sqlite3
-   con = sqlite3.connect('example.db')
+   con = sqlite3.connect("tutorial.db")
 
-The special path name ``:memory:`` can be provided to create a temporary
-database in RAM.
+The returned :class:`Connection` object ``con``
+represents the connection to the on-disk database.
 
-Once a :class:`Connection` has been established, create a :class:`Cursor` object
-and call its :meth:`~Cursor.execute` method to perform SQL commands::
+In order to execute SQL statements and fetch results from SQL queries,
+use a database cursor.
+Call :meth:`con.cursor() <Connection.cursor>` to create the :class:`Cursor`::
 
    cur = con.cursor()
 
-   # Create table
-   cur.execute('''CREATE TABLE stocks
-                  (date text, trans text, symbol text, qty real, price real)''')
+Now, create a database table ``movie`` with columns for title,
+release year, and review score.
+The `flexible typing`_ feature of SQLite makes typenames optional;
+for simplicity, just use column names in the table declaration.
```

Comment:
```suggestion
For simplicity, just use column names in the table declaration:
thanks to the `flexible typing`_ feature of SQLite, specifying the types is optional.
```
This suggestion uses a simpler terminology since `typenames` hasn't been introduced yet.

Reply:
That's better. Perhaps "data types" iso. "types".